mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -821,13 +821,15 @@ public void onSuccess(Call call, int statusCode, Headers headers, byte[] respons
     private void showSitemapSelectionDialog(final List<OpenHABSitemap> sitemapList) {
         Log.d(TAG, "Opening sitemap selection dialog");
         final List<String> sitemapNameList = new ArrayList<String>();
+        final List<String> sitemapLabelList = new ArrayList<String>();
         for (int i = 0; i < sitemapList.size(); i++) {
             sitemapNameList.add(sitemapList.get(i).getName());
```

Comment:
is it really needed to have two lists here? Wouldn't it be possible to get rid of sitemapNameList? When accessing the sitemapNameList in the dialog, you could access sitemapList, get the element at the position of "item" and execute getName on that.

If we would've API Level 24 as the minimum supported, we could even use streams and wouldn't need any additional list here (at least not as a local variable).